frame/mosaic/README.md

```diff
@@ -0,0 +1,10 @@
+# Mosaic
```

Comment:
please add referene to mosic docs so we can check if properly handles txs - avoids double mints and double burns.

Reply:
CU-1wanc6n